.travis.yml

```diff
@@ -22,6 +22,7 @@ addons:
       - perl
       - cpanminus
       - etcd
+      - kong/kong/luarocks
```

Comment:
why we use `kong/luarocks`? it works fine to use `luarocks` at my local machine.